docs/scripting/syntax/colors.mdx

```diff
@@ -107,3 +107,4 @@ They are detailed below, with a description and the color identifier.
 | ((%D:::TRUST COMMUNICATION:::%))                                                                                                 | Trust communications use the color ((%DD%)).                                                                                                                                                                                                                                     |
 | ((%C@%))((%Juser%)), ((%C@%))((%Kuser%)), ((%C@%))((%Muser%)), ((%C@%))((%Wuser%)), ((%C@%))((%Luser%)), and ((%C@%))((%Buser%)) | Usernames can use six different colors. The ((%C@%)) uses color ((%CC%)), and the name itself can take on any of the colors JKMWLB.                                                                                                                                              |
 | ((%A3019%))((%BAD%)) ((%CD%))((%L123%))                                                                                          | This date format has four elements. The year is highlighted with color ((%AA%)), the day with color ((%LL%)). "((%BAD%))" is colored with color ((%BB%)), and "((%CD%))" is colored with color ((%CC%)).                                                                         |
+| Upgrade rarities ((%Cnoob%)) ,((%Akiddie%)) ,((%Lh4x0r%)) ,((%Ph4rdc0r3%)) ,((%T\|\_\|b3\|2%)) ,and ((%F31337%))                 | Upgrade rarities use the colors ((%CC%)), ((%AA%)), ((%LL%)), ((%PP%)), ((%TT%)), and ((%FF%)) respectively.                                                                                                                                                                     |
```

Comment:
```suggestion
| Upgrade rarities ((%0noob%)), ((%1kiddie%)), ((%2h4x0r%)), ((%3h4rdc0r3%)), ((%4\|\_\|b3\|2%)), and ((%531337%))                 | Upgrade rarities are colored using the rarity value as a number: ((%00%)), ((%11%)), ((%22%)), ((%33%)), ((%44%)), and ((%55%)) respectively.                                                                                                                                    |
```
This is definitely a good idea to document, but I think we should go with the number version to make more clear the relationship between the rarity value and the color.

Reply:
(I also moved some commas to the appropriate place in the suggestions.)

Reply:
Oh, good to know! I didn't realize the number codes were even set up in this way. In this case, I agree.

Reply:
You can use the buttons under the suggestion to make the commit then.